index.js

```diff
@@ -156,6 +168,34 @@ async function getActionId (code) {
   return row ? row.id : null
 }
 
+async function getAllDataForDownload (code) {
+  return query(`SELECT 
+    to_char(timestamp::date,'DD/MM/YYYY') AS date,  
+    timestamp::time(0) AS time,  
+    company.name AS company, 
+    pseudonym, 
+    action.code, 
+    action.description
+  FROM action_log
+  INNER JOIN company ON action_log.company_id=company.id
+  INNER JOIN action ON action_log.action_id = action.id;`)

```

Comment:
Inconsistent spacing around the `=` signs. Change to version with space padding, i.e.:
```
action_log.company_id = company.id
```
Same applies below.